### PR TITLE
Delete file share specification

### PR DESCRIPTION
**Pull request by kamleshverma1**

### CODE REVIEW

**Comment by kamleshverma1** [comment]
@gnehapk @cloudbehl Please review it.

File: specs/delete-file-share-spec.adoc [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+        ** Cluster object details -> File share tab - delete action.
+
+        ** File share object details - delete action
+
+    - Delete file share wizard will be modal popup, which will populate following information:
+
+    	** Warning message to draw attention to the fact that this file share will be permanently deleted.
+
+    	** And provide current utilization , a list of clients accessing this file share and a summary of bricks and usage.
```
Comment (kamleshverma1) [inline specs/delete-file-share-spec.adoc:19]: @anivargi @shtripat @nnDarshan Can i have some sample response data for file share's utilization and summary of bricks with usage for specific file share.

And i guess , there is no support for connected clients as of now , correct me if i am wrong .

Comment (shtripat) [reply specs/delete-file-share-spec.adoc:19]: That's correct. No support for connected clients at the moment. Added the sample volume listing data from api server below 

```
[

    {
        "123e0a4d-317b-4229-91dc-ea6a1c78819f": {
            "arbiter_count": "0",
            "usable_capacity": "",
            "name": "vol1",
            "rebal_id": "8969c312-ea97-4aa4-8c52-fb452c551098",
            "rebal_status": "completed",
            "transport_type": "tcp",
            "vol_type": "Distribute",
            "snap_count": "0",
            "rebal_skipped": "0",
            "snapd_inited": "True",
            "snapd_status": "Offline",
            "brick_count": "2",
            "disperse_count": "0",
            "rebal_data": "0Bytes",
            "rebal_failures": "0",
            "rebal_lookedup": "0",
            "stripe_count": "1",
            "subvol_count": "2",
            "deleted": "",
            "redundancy_count": "0",
            "replica_count": "1",
            "vol_id": "123e0a4d-317b-4229-91dc-ea6a1c78819f",
            "rebal_files": "0",
            "bricks": {
                "dhcp43-109.lab.eng.blr.redhat.com:_root_bricks_vol1_b1": {
                    "filesystem_type": "",
                    "hostname": "dhcp43-109.lab.eng.blr.redhat.com",
                    "mount_opts": "",
                    "path": "dhcp43-109.lab.eng.blr.redhat.com:/root/bricks/vol1_b1",
                    "port": "49152",
                    "status": "Started",
                    "vol_id": "123e0a4d-317b-4229-91dc-ea6a1c78819f"
                },
                "dhcp43-49.lab.eng.blr.redhat.com:_root_bricks_vol1_b2": {
                    "filesystem_type": "",
                    "hostname": "dhcp43-49.lab.eng.blr.redhat.com",
                    "mount_opts": "",
                    "path": "dhcp43-49.lab.eng.blr.redhat.com:/root/bricks/vol1_b2",
                    "port": "",
                    "status": "",
                    "vol_id": "123e0a4d-317b-4229-91dc-ea6a1c78819f"
                }
            },
            "pcnt_used": "",
            "used_capacity": "",
            "quorum_status": "not_applicable",
            "status": "Started",
            "options": {
                "performance.readdir-ahead": "on",
                "nfs.disable": "on",
                "transport.address-family": "inet"
            }
        }
    },
    {
        "96c6169a-3135-4653-90b6-ca8a339bc4f5": {
            "brick_count": "2",
            "rebal_failures": "0",
            "redundancy_count": "0",
            "status": "Started",
            "rebal_id": "c7b8af98-202f-45e5-829b-0f3f2d959ae9",
            "rebal_skipped": "0",
            "replica_count": "1",
            "vol_id": "96c6169a-3135-4653-90b6-ca8a339bc4f5",
            "pcnt_used": "",
            "usable_capacity": "",
            "deleted": "",
            "disperse_count": "0",
            "snapd_inited": "True",
            "transport_type": "tcp",
            "name": "vol2",
            "snap_count": "0",
            "bricks": {
                "dhcp43-109.lab.eng.blr.redhat.com:_root_bricks_vol2_b1": {
                    "status": "Started",
                    "vol_id": "96c6169a-3135-4653-90b6-ca8a339bc4f5",
                    "filesystem_type": "",
                    "hostname": "dhcp43-109.lab.eng.blr.redhat.com",
                    "mount_opts": "",
                    "path": "dhcp43-109.lab.eng.blr.redhat.com:/root/bricks/vol2_b1",
                    "port": "49153"
                },
                "dhcp43-49.lab.eng.blr.redhat.com:_root_bricks_vol2_b2": {
                    "status": "",
                    "vol_id": "96c6169a-3135-4653-90b6-ca8a339bc4f5",
                    "filesystem_type": "",
                    "hostname": "dhcp43-49.lab.eng.blr.redhat.com",
                    "mount_opts": "",
                    "path": "dhcp43-49.lab.eng.blr.redhat.com:/root/bricks/vol2_b2",
                    "port": ""
                }
            },
            "arbiter_count": "0",
            "rebal_data": "0Bytes",
            "vol_type": "Distribute",
            "options": {
                "performance.readdir-ahead": "on",
                "transport.address-family": "inet",
                "nfs.disable": "on"
            },
            "stripe_count": "1",
            "subvol_count": "2",
            "quorum_status": "not_applicable",
            "rebal_files": "0",
            "rebal_lookedup": "0",
            "rebal_status": "completed",
            "snapd_status": "Offline",
            "used_capacity": ""
        }
    }
]
```

All the required fields are available here (data could be wrong as its from a dev setup).

Comment (kamleshverma1) [reply specs/delete-file-share-spec.adoc:19]: @shtripat Thanks for quick response

**Comment by r0h4n** [comment]
@cloudbehl please take over this one in a separate spec, closing this since the owner isnt an active member

**Pull request closed without merging**